src/Basics/Extra.elm

```diff
@@ -37,6 +39,11 @@ module Basics.Extra exposing
 
 @docs flip, curry, uncurry
 
+
+# Comparison & Ordering
+
+@docs orderBy, toOrder
```

Comment:
There is a @docs entry missing for `toOrderDesc`

Reply:
Good catch, thank you!